src/sentry/notifications/notifications/rules.py

```diff
@@ -151,6 +152,19 @@ def send(self) -> None:
         shared_context = self.get_context()
 
         for provider, participants in participants_by_provider.items():
+            if self.target_type == ActionTargetType.RELEASE_MEMBERS:
+                last_release = shared_context["last_release"]
```

Comment:
nit: Might be safer to use `shared_context.get("last_release", None)` instead of keying on the context.